diagnostic.md

```diff
@@ -94,4 +92,4 @@ var teamMember = {
 What is the value of `teamMember['hobby']`?  Of `teamMember['name']`?  Of
 `teamMember['name']['surname']`?
 
-_Replace this text with your answer._
+The above does not appear to be a valid expression.  For example, if we wanted to know the value of "role", the expression would be "teamMember.role;"  The expression as written above would not return a value.
```

Comment:
What you are referring to is dot notation. This question uses bracket notation, which is another way that we can access a property of an object. The answers would be as follows: 

```
 teamMember['hobby'] = 'exercise';
 
 teamMember['name'] = {
   given: 'Jason',
   surname: 'Weeks'
 };
 
 teamMember['name']['surname'] = 'Weeks';
 ```